<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbxCntrl">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>221</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Avatar Cotrol</string>
    </property>
    <widget class="QPushButton" name="btnMoveLeft">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMoveRight">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMoveUp">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>/\</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMoveDown">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>\/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLetGrow">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLetShrink">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnResetPosition">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Position
zurücksetzen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnResetSize">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Größe
zurücksetzen</string>
     </property>
    </widget>
    <widget class="QSlider" name="sldRed">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>250</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>85</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSlider" name="sldGreen">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>280</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>170</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSlider" name="sldBlue">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>310</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblRed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rot: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblGreen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Grün: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblBlue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Blau:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblClr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Farbe: rgb(85, 170, 0)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRand">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>340</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Zufällig</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblElementInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>lblElementInfo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAvatar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
